app/scripts/services/spouse.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+angular.module('confRegistrationWebApp')
+  .factory('spouse', function ($http, $cacheFactory) {
+    var spouseRegistrationCache = $cacheFactory('spouseRegistration');
+
+    var spouse = {
+      // Preload a spouse registration so that calling getSpouseRegistration for that conference will complete sooner
+      // This functionality is provided because looking up a spouse registration via getSpouseRegistration is a lengthy operation
+      preloadSpouseRegistration: function (conferenceId) {
+        return spouse.getSpouseRegistration(conferenceId);
```

Comment:
My thought would be to make it 2 functions when there are 2 different caching strategies. Or at least add a TODO comment if you are intending to make those changes sometime soon. I think it just makes it confusing to have comments saying the functions are different when in reality they aren't. Idk the bigger picture so I'll let you decide what to do with them.